app/templates/services/_base_service_page.html

```diff
@@ -1,6 +1,6 @@
 {% extends "_base_page.html" %}
 
-{% block page_title %}{{ service_data.serviceName or service_data.lotName }} – Digital Marketplace{% endblock %}
+{% block page_title %}{{ framework.name }} {{ service_data.lotName }} – Digital Marketplace{% endblock %}
```

Comment:
1. For editing existing services, keep as before.
2. Something generic for new services, like ‘Summary – Digital Marketplace’ or ‘Overview – Digital Marketplace’.